README.md

```diff
@@ -78,6 +79,57 @@ This document describes how to deploy ui-bakery on-prem via `install.sh` script.
 
 **NOTE**: If Docker of the version less than the required (minimum 20.10.11) is already installed on the server, and/or Docker Compose (minimum 1.29.2), the script will be stopped. You need to update the versions of components manually and run the script again.
 
+#### Features of installation on Azure Virtual machine with Ubuntu
+
+1. Open [Azure Portal](http://portal.azure.com/) and on the search field in the top type in `Virtual machines` and select this item in result search window
+
+1. Click on `+ Create` button in the top left corner and select `Azure Virtual machine` in the menu
+
+1. Select an image of `Ubuntu 18.04` or higher
+
+1. For instance size, select minimum `Standard_F2s_v2 - 2 vCPUs, 4 GiB memory`
+
+1. If you want to access through password, in `Administrator account` section in `Authentication type` menu item select `Password` and setup admin username and password. Otherwise leave `SSH public key` and it will generate the keys after creation
+
+1. In `Inbound port rules` leave `SSH(22)` as by default
```

Comment:
Don't you think we have to open other ports to allow http/https traffic?

Reply:
I tested it with only Bakery and SSH ports open and it worked